mlflow/spark.py

```diff
@@ -360,12 +363,15 @@ def save_model(spark_model, path, mlflow_model=Model(), conda_env=None,
                          This must be a PySpark DataFrame that the model can evaluate. If
                          ``sample_input`` is ``None``, the MLeap flavor is not added.
 
-    >>> from mlflow import spark
-    >>> from pyspark.ml.pipeline.PipelineModel
-    >>>
-    >>> #your pyspark.ml.pipeline.PipelineModel type
-    >>> model = ...
-    >>> mlflow.spark.save_model(model, "spark-model")
+    .. code-block:: python
+        :caption: Example
+
+        from mlflow import spark
+        from pyspark.ml.pipeline.PipelineModel
+
+        #your pyspark.ml.pipeline.PipelineModel type
```

Comment:
Add a space after the hash symbol?

Reply:
Good catch. Will fix them and do a commit shortly

Reply:
Fixed!